docs/conf.py

```diff
@@ -1,78 +1,66 @@
-    "nengo_sphinx_theme.ext.versions",
+    "sphinx.ext.viewcode",
+    "nengo_sphinx_theme",
+    "numpydoc",
+    "nbsphinx",
 ]
 
-# -- sphinx
-nitpicky = True
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
-source_suffix = ".rst"
-source_encoding = "utf-8"
-master_doc = "index"
-project = "{{ PROJECT }}"
-copyright = proj.__copyright__
-author = "Applied Brain Research"
-version = ".".join(proj.__version__.split(".")[:2])  # X.Y version
-release = proj.__version__ # Full version with tags
+templates_path = ["_templates"]
+
+# -- sphinx.ext.autodoc
+autoclass_content = "both"  # class and __init__ docstrings are concatenated
+autodoc_default_options = {"members": None}
+autodoc_member_order = "bysource"  # default is alphabetical
 
 # -- sphinx.ext.intersphinx
 intersphinx_mapping = {
-    "nengo": ("https://www.nengo.ai/", None)
+    "numpy": ("https://docs.scipy.org/doc/numpy", None),
+    "nengo": ("https://www.nengo.ai/nengo/", None),
 }
 
-# -- sphinx.ext.todo
-todo_include_todos = True
-
-# -- nengo_sphinx_theme
-html_theme = "nengo_sphinx_theme"
-pygments_style = "friendly"
-templates_path = []
-html_favicon = ""
-html_static_path = ["_static"]
-html_logo = os.path.join("_static", "logo.svg")
-html_sidebars = {"**": ["sidebar.html"]}
-html_context = {
-    "css_files": [os.path.join("_static", "custom.css")],
-}
+# -- numpydoc config
+numpydoc_show_class_members = False
 
-# -- other
-htmlhelp_basename = project
+# -- nbsphinx
+nbsphinx_timeout = 300
 
-latex_elements = {
-    # "papersize": "letterpaper",
-    # "pointsize": "11pt",
-    # "preamble": "",
-    # "figure_align": "htbp",
-}
+# -- sphinx
+exclude_patterns = ["_build", "**.ipynb_checkpoints"]
+source_suffix = ".rst"
+source_encoding = "utf-8"
+master_doc = "index"
+suppress_warnings = ["image.nonlocal_uri"]
+linkcheck_ignore = [r"http://localhost:\d+"]
+linkcheck_anchors = True
+nitpicky = True
+default_role = "py:obj"
 
-latex_documents = [
-    (master_doc,  # source start file
-     "{{ SHORT PROJECT }}.tex",  # target name
-     project,  # title
-     author,  # author
-     "manual"),  # documentclass
-]
+project = u"Nengo Bones"
+authors = u"Applied Brain Research"
+copyright = nengo_bones.__copyright__
+release = nengo_bones.__version__  # Full version, with tags
+pygments_style = "friendly"
 
-man_pages = [
-    # (source start file, name, description, authors, manual section).
-    (master_doc, "{{ SHORT PROJECT }}", project, [author], 1)
-]
+# -- Options for HTML output --------------------------------------------------
 
-texinfo_documents = [
-    (master_doc,  # source start file
-     "{{ SHORT PROJECT }}",  # target name
-     project,  # title
-     author,  # author
-     "{{ SHORT PROJECT }}",  # dir menu entry
-     "{{ DESCRIPTION }}",  # description
-     "Miscellaneous"),  # category
-]
+html_theme = "nengo_sphinx_theme"
+html_title = "Nengo Bones documentation"
+# html_static_path = ["_static"]
+htmlhelp_basename = "NengoBonesdoc"
```

Comment:
Make consistent with other repos? e.g., in Nengo
```python
html_title = "Nengo core {0} docs".format(release)
htmlhelp_basename = 'Nengo core'
```

Reply:
Yeah it should be.  I'm also hoping that this is one of the files we can start templating, and then this will all be nice and consistent!

Reply:
Done